benchopt/cli/helpers.py

```diff
@@ -40,6 +44,141 @@ def clean(benchmark, token=None, filename=None):
+
+
+@helpers.command(
+    help="List information and requirements (solvers/datasets) "
+    "for a given benchmark.",
+    epilog="To (re-)install the required solvers and datasets "
+    "in a benchmark-dedicated conda environment or in your own "
+    "conda environment, see the command `benchopt install`."
+)
+@click.argument('benchmark', type=click.Path(exists=True),
+                autocompletion=complete_benchmarks)
+@click.option('--env', '-e', 'env_name',
+              flag_value='True', type=str, default='False',
+              help="Additional checks for requirement availability in "
+              "the dedicated conda environment for the benchmark "
+              "named 'benchopt_<BENCHMARK>'.")
+@click.option('--env-name', 'env_name',
+              metavar="<env_name>", type=str, default='False',
+              shell_complete=complete_conda_envs,
+              help="Additional checks for requirement availability in "
+              "the conda environment named <env_name>.")
+def info(benchmark, env_name):
+
+    # benchmark
+    benchmark = Benchmark(benchmark)
+    print(f"Info regarding '{benchmark.name}'")
+
+    # get solvers and datasets in the benchmark
+    solvers = benchmark.get_solvers()
+    datasets = benchmark.get_datasets()
+
+    # Get a list of all conda envs
+    default_conda_env, conda_envs = list_conda_envs()
+
+    # Check conda env (if relevant)
+
+    # If env_name is False (default), check availability
+    # in the current environement.
+    if env_name == 'False':
+        # check if any current conda environment
+        if default_conda_env is not None:
+            env_name = default_conda_env
+        else:
+            raise RuntimeError("No conda environment is activated.")
+    else:
+        # If env_name is True, the flag `--env` has been used. Check the conda
+        # env dedicated to the benchmark. Else, use the <env_name> value.
+        if env_name == 'True':
+            env_name = f"benchopt_{benchmark.name}"
+        else:
+            # check provided <env_name>
+            # (to avoid empty name like `--env-name ""`)
+            if len(env_name) == 0:
+                raise RuntimeError("Empty environment name.")
+            if env_name not in conda_envs:
+                raise RuntimeError(
+                    f"{env_name} is not an existing conda environment."
+                )
+
+    # check env_name environment
+    check_benchopt = _run_shell_in_conda_env(
+        "benchopt --version", env_name=env_name, capture_stdout=True
+    )
+    if check_benchopt != 0:
+        msg = f"!! Environment '{env_name}' does not exist " + \
+            "or is not configurated for benchopt, " + \
+            "benchmark requirement availability will not be checked, " + \
+            "see the command `benchopt install`."
+        print(msg)
+        env_name = None
+    else:
+        msg = "Checking benchamrk requirement availability " + \
+            f"in env '{env_name}'."
```

Comment:
I would say avoid `\` as much as possible. I find it hard to read in the code.
```suggestion
        msg = (
            "Checking benchamrk requirement availability "
            f"in env '{env_name}'."
        )
```

Reply:
ok, no problem, done.

Reply:
thx!